lib/danger/ci_source/circle_api.rb

```diff
@@ -45,7 +45,7 @@ def fetch_build(repo_slug, build_number, token)
       url = "project/#{repo_slug}/#{build_number}"
       params = { "circle-token" => token }
       response = client.get url, params, accept: "application/json"
-      JSON.parse(response.body, symbolize_names: true)
+      JSON.parse(response.body, symbolize_names: true, encoding: Encoding::UTF_8)
```

Comment:
I'm afraid that `JSON.parse()` does not have option `encoding`.
https://docs.ruby-lang.org/en/master/JSON.html#module-JSON-label-Parsing+Options

The workaround described in ruby/json adds `encoding` option to `File.read()`.
https://github.com/ruby/json/issues/697#issuecomment-2462442823